### DISCUSSION TITLE
Best option for a timepicker

### DISCUSSION THREAD

**Discussion by lostmarinero** [chakra-ui/chakra-ui#8751] [Q&A] [answered]
I need to use a timepicker, and I was wondering what you all have done in similar situation.

I'd use the built in <Input type="time />, but i need to have it be increments of 30 minutes and the 'steps' attribute does not work across all broswers.

Any thoughts/ideas?

**Comment by segunadebayo** [comment, marked as answer]
There is no TimePicker. Use `Combobox` with 30-minute items, or a native `input type="time"` where `step` is enough.

DatePicker covers dates, not clock times.
